### DISCUSSION TITLE
LTDC setup, glitched rendering

### DISCUSSION THREAD

**Discussion by epikao** [stm32duino/Arduino_Core_STM32#1598] [Q&A]
Hello,

your LTDC setup (from fork / branch:https://github.com/fpistm/Arduino_Core_STM32/tree/LTDC_TS_TEST_1.9.0/libraries) works so far very good, I only have an issue with glitched rendering.

According https://github.com/lvgl/lvgl/issues/2903, it looks it has something to do with CPU cache is not being invalidated at the right time or isn't configured correctly....

Do you have any idea, improvement/ adjustments for your LTDC files?

Thank you

**Comment by fpistm** [comment]
Hi @epikao 
I've converted your issue as a GH discussion as it is not relevant for the core. This branch was provided "as it".
You're right it is probably linked to the cache. It has already been discussed on the forum:
https://www.stm32duino.com/viewtopic.php?p=4924#p4924
